plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -226,14 +227,39 @@ public MetadataEntry getMetadataEntry(TableSnapshot tableSnapshot, ConnectorSess
     @Deprecated
     public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, ConnectorSession session)
     {
-        return getActiveFiles(tableSnapshot, metadataEntry, protocolEntry, TupleDomain.all(), session);
+        return retrieveActiveFiles(tableSnapshot, metadataEntry, protocolEntry, TupleDomain.all(), Optional.empty(), session);
     }
 
-    public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, TupleDomain<DeltaLakeColumnHandle> partitionConstraint, ConnectorSession session)
+    public List<AddFileEntry> getActiveFiles(
+            TableSnapshot tableSnapshot,
+            MetadataEntry metadataEntry,
+            ProtocolEntry protocolEntry,
+            TupleDomain<DeltaLakeColumnHandle> partitionConstraint,
+            Optional<Set<DeltaLakeColumnHandle>> projectedColumns,
+            ConnectorSession session)
+    {
+        Optional<Function<String, Boolean>> addStatsColumnFilter = Optional.empty();
+        if (projectedColumns.isPresent()) {
+            Set<String> baseColumnNames = projectedColumns.get().stream()
+                    .filter(DeltaLakeColumnHandle::isBaseColumn) // Only base column stats are supported
+                    .map(DeltaLakeColumnHandle::getColumnName)
```

Comment:
Normalized (lowercased) or not?

Reply:
We are using here the original column names - e.g `a_NuMbEr` instead of `a_number`.

See corresponding test `io.trino.plugin.deltalake.TestDeltaLakeBasic#testCheckpointFilteringForParsedStatsWithCaseSensitiveColumnNames` and test resource: `databricks133/parsed_stats_case_sensitive/_delta_log/00000000000000000002.checkpoint.parquet`

```
    optional group stats_parsed {
      optional int64 numRecords;
      optional group minValues {
        optional int32 a_NuMbEr;
        optional binary a_StRiNg (STRING);
      }
```